timeseries/src/autogluon/timeseries/models/local/abstract_local_model.py

```diff
@@ -113,8 +113,11 @@ def _fit(self, train_data: TimeSeriesDataFrame, time_limit: Optional[int] = None
         local_model_args = {}
         # TODO: Move filtering logic to AbstractTimeSeriesModel
         for key, value in raw_local_model_args.items():
-            if key in self.allowed_hyperparameters:
+            if key in self.allowed_local_model_args:
```

Comment:
was this a bug?

Reply:
Yes, but this wasn't caught because we didn't have tests that added `target_scaler` to all models. Will fix that in this PR.